ReleaseNotes

```diff
@@ -31615,3 +31615,894 @@ All boards in the NuttX tree have been updated.
 NuttX users developing their own custom boards will need to remove any #include "arm_arch.h" and (if necessary), #include "arm_internal.h" instead.
 
 See PR-5720, git commit 54e630e14d7e32d6f81ae79d4e5df3d2fa787ef0.
+
+NuttX-11.0.0 Release Notes
+
+# What's New In This Release
+
+## Bug Fixes and Improvements
+
+ * [#5978](https://github.com/apache/incubator-nuttx/pull/#5978) nuttx: unify FAR attribute usage across the code
```

Comment:
I have a simple question and add an issue https://github.com/apache/incubator-nuttx/issues/7008.
The correct link to a pull request is `https://github.com/apache/incubator-nuttx/pull/5978`.